Makefile

```diff
@@ -4,12 +4,12 @@ IMAGE_NAME ?= cloud-resource-operator
 OPERATOR_IMG = $(IMAGE_REG)/$(IMAGE_ORG)/$(IMAGE_NAME):v$(VERSION)
 MANIFEST_NAME ?= cloud-resources
 NAMESPACE=cloud-resource-operator
-PREV_VERSION=0.39.0
-VERSION=0.40.0
+PREV_VERSION=0.40.0
```

Comment:
@laurafitzgerald, that's correct. Instead of doing a separate PR to rebase the feature branch with master, I pulled the commits from master to this branch, together with the work on the current JIRA. Is that okay?

Reply:
yeah that's fine. it's a bit easier to review when rebasing but i think it's fine this time. are you ready to go ahead and merge this pr then?

Reply:
It's ready to be merged 👍🏻 Just needs to be approved. Would you mind posting your approval?